packages/eslint-config-typescript/index.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
+  parser: '@typescript-eslint/parser',
+  extends: [
+    '@zonedigital/eslint-config-zone',
+    'plugin:import/typescript',
+    'plugin:@typescript-eslint/eslint-recommended',
+    'plugin:@typescript-eslint/recommended',
+    'plugin:prettier/recommended',
+    'prettier/@typescript-eslint',
+  ],
+  plugins: ['@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/explicit-function-return-type': 'off',
```

Comment:
An approach we have taken elsewhere is to only add type annotations when TS can’t infer the type, kinda like TS as a progressive enhancement 🤔 However I think this might be project dependent. Happy to remove this for now, it’s likely to cause more confusion than anything.